dcp-0010/dcp-0010.mediawiki

```diff
@@ -0,0 +1,449 @@
+<pre>
+DCP: 0010
+Title: Change PoW/PoS Subsidy Split To 10/80
+Author: Dave Collins <davec@decred.org>
+        Jake Yocom-Piatt
+Status: Defined
+Created: 2021-12-16
+License: CC0-1.0
+License-Code: ISC
+Requires: DCP0006
```

Comment:
Do you think it's worth mentioning why this requires DCP0006?  It doesn't matter much at this point since it's already activated deep in the chain, but it doesn't seem like this is a hard dependency other than it was another proposal that impacted where part of the subsidy was directed (and, for example, the reference implementation for this proposal would still work fine whether DCP0006 passed or not).

Reply:
I can add a note to the subsidy enforcement section.  The reason I added it as a requirement is because the specification of the rules that must apply to the coinbase assume DCP0006 is active.

Reply:
I should also note that this is also why the following bit is there:

"The Treasury subsidy proportion MUST remain the same and otherwise retain all existing semantics it has prior to this specification."

Reply:
Makes sense, thanks for the clarification.

Reply:
Add a sub section to describe it.